File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.java
class Solution {
    public int minMoves2(int[] nums) {
        int mean = 0;
        int ans = 0;
        Arrays.sort(nums);
        mean = nums[nums.length/2];
        for(int num : nums)
            ans+=Math.abs(mean-num);
        return ans;
    }
}

File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.cpp
class Solution {
public:
    int minMoves2(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int i=0,j=nums.size()-1;
        int ans = 0;
        while(i<j)
        {
            ans += nums[j]-nums[i];
            i++;
            j--;
        }
        return ans;
    }
};